lms/validation/_base.py

```diff
@@ -136,14 +137,18 @@ def parse(self, *args, **kwargs):
         either return the successfully parsed params or raise a validation
         error.
 
-        :raise lms.validation.ValidationError: if the response isn't valid
+        :raise ExternalRequestError: if the response isn't valid
         """
         try:
-            result = self.load(self.context["response"], *args, **kwargs)
+            return self.load(self.context["response"], *args, **kwargs)
         except marshmallow.ValidationError as err:
-            raise ValidationError(messages=err.messages) from err
+            response = self.context.get("response")
+            request = response.request if response is not None else None
+            validation_errors = err.messages
 
-        return result
+            raise ExternalRequestError(
+                request=request, response=response, validation_errors=validation_errors
+            ) from err
```

Comment:
This is where I changed `RequestsResponseSchema` to raise an `ExternalRequestError` with the `request`, `response` and `validation_errors` instead of raising a `ValidationError`. The `ExternalRequestError` will get caught by the `external_request_error()` exception view that reports the `request`, `response` and `validation_errors` to Sentry and the frontend (if nothing in the services or views catches and handles the exception).